xml/snmp_sysdescr.xml

```diff
@@ -2314,7 +2314,7 @@ Copyright (c) 1995-2005 by Cisco Systems
    =======================================================================-->
 
    <!-- Sneaky. The XOS uses both Zero and Oh depending on version -->
-   <fingerprint pattern="Extreme(?:X[0O]S|Ware X[0O]S) version (\S+) \S+ by \S+ on .*$">
+   <fingerprint pattern="Extreme(?:X[0O]S|Ware X[0O]S) (?:\s?\([Xx]\d{0,3}-\d{0,2}[Xx]\)\s)?version (\S+) \S+ by \S+ on .*$">
```

Comment:
This might be too narrow and may result in missed matches, but I don't have any other examples to go on right now.